src/r_draw.c

```diff
@@ -526,21 +526,181 @@ static void R_DrawFuzzColumn_block(void)
     }
 }
 
-// [FG] spectre drawing mode: 0 original, 1 blocky (hires)
+#define FUZZDARK 6 * 256
 
-boolean fuzzcolumn_mode;
-void (*R_DrawFuzzColumn)(void) = R_DrawFuzzColumn_orig;
+static const int fuzzdark[FUZZTABLE] =
+{
+    FUZZDARK,0,FUZZDARK,0,FUZZDARK,FUZZDARK,0,
+    FUZZDARK,FUZZDARK,0,FUZZDARK,FUZZDARK,FUZZDARK,0,
+    FUZZDARK,FUZZDARK,FUZZDARK,0,0,0,0,
+    FUZZDARK,0,0,FUZZDARK,FUZZDARK,FUZZDARK,FUZZDARK,0,
+    FUZZDARK,0,FUZZDARK,FUZZDARK,0,0,FUZZDARK,
+    FUZZDARK,0,0,0,0,FUZZDARK,FUZZDARK,
+    FUZZDARK,FUZZDARK,0,FUZZDARK,FUZZDARK,0,FUZZDARK
+};
+
+static void DrawFuzzColumnRefraction(void)
```

Comment:
Refraction looks pretty bad when there's a solid color background. See fuzz3.wad:

![woof0003](https://github.com/user-attachments/assets/f7b310b9-33af-4362-b640-9c8767ac1ea4)

When I added this feature to Nugget Doom, I applied some noise to the effect and the profile/outline is wavy, which looks better to me. I'm not entirely satisfied with the code (too much branching, looks worse at higher framerates), so please don't use the exact method, but something similar would look nice.

https://github.com/user-attachments/assets/f115a1fb-0cae-4a42-acf0-d5523ff0f847

Reply:
I guess we can generate another table for noise to eliminate branching.

Reply:
Done in https://github.com/fabiangreffrath/woof/pull/2043/commits/b7e4b7ae712b95dbf05b265c9d56cd01928bd247

I generated the `fuzzdark` table using this code from Nugget:
```c
#define FUZZDARK (256 * (Woof_Random() < 32 ? (Woof_Random() & 1 ? 4 : 8) : 6))
#define FUZZSELECT (fuzzoffset[fuzzpos] ? 0 : FUZZDARK)
```

Reply:
Looks good, I like it!

Reply:
Okay, that looks good.